### ISSUE TITLE
Update plugin to support homebridge v2

### ISSUE THREAD

**Issue by LeLunZ** [smhex/homebridge-wled-ws#145]
With the newest homebridge version you now get a warning (that's not plugin specific):

```
NOTICE TO USERS AND PLUGIN DEVELOPERS
> Homebridge 2.0 is on the way and brings some breaking changes to existing plugins.
> Please visit the following link to learn more about the changes and how to prepare:
> https://github.com/homebridge/homebridge/wiki/Updating-To-Homebridge-v2.0
```

I currently can't test the plugin with v2 beta, because my esp isn't working :/ 

---

Also: With the newest version they have now also implemented a warning for characters in device names which can cause errors.
In a plugin I help to maintain we use [this function](https://github.com/MrAsterisco/homebridge-hisense-tv/blob/master/src/utils/validateHomeKitName.function.ts) and just wrap all names/configuredNames.

**Comment by smhex** [comment]
I have seen those warnings too. However, I do not have a v2 beta up and running. Regarding the name functionality: where in the code would you implement it? It could be part of the configuration scheme...

**Comment by LeLunZ** [comment]
If you think about creating a scheme which a user has to follow when configuring the plugin, I am not 100% sure, If I would do it like that. Because for example with the Presets, if a user already has Presets configured in Wled, and now also needs to change the naming of them (to get them in sync with homebridge-wled-ws) it would be a bit of a hassle.

But I think the issue can just be resolved when stripping all unnecessary characters the initialisation step of a Service.
Because you aren't really accessing the service anymore, because you got the actual name stored in your WledControllerPreset object.

I am actually not sure about the ConfiguredName (Characteristic) and displayName (second argument in the constructor). I will check that and see if those are also affected.

**Comment by smhex** [comment]
Yes, that sounds reasonable. Changing all the preset names is not the best option. However, the usage of the ConfiguredName property led to different results during my tests. My assumption is that Apple changed the something over the time...

**Comment by mishakim** [comment]
Any update on the compatibility?

**Comment by smhex** [comment]
@mishakim : 0.17.0-beta.0 with Homebridge v2 support published
